searx/plugins/limiter.py

```diff
@@ -40,6 +42,39 @@
     + r')'
 )
 
+PING_KEY = 'SearXNG_limiter.ping'
+TOKEN_KEY = 'SearXNG_limiter.token'
+
+
+def ping():
+    redis_client = redisdb.client()
+    user_agent = request.headers.get('User-Agent', 'unknown')
+    x_forwarded_for = request.headers.get('X-Forwarded-For', '')
```

Comment:
what happen when `X-Forwarded-For` is empty like for local usage with a request directly to searxng without passing through a reverse proxy?